src/main/java/shon/Shon.java

```diff
@@ -7,22 +7,32 @@
 import shon.exception.CommandException;
 import shon.exception.ParameterException;
 
+/**
+ * Represents a chatbot that maintains a Todo List.
+ */
 public class Shon {
-
     private Ui ui;
```

Comment:
Just curious, why not make `Ui` static like the `Parser` class? Was it a purposeful design choice?